lib/tasks/e2e-test.js

```diff
@@ -85,32 +83,29 @@ const configure = function configure (gulp, opts, env) {
       ]);
     }
 
-    // go through the steps to run the test
-    return startupSeq.reduce((promise, step) => promise.then(step), B.resolve())
-      .then(function () {
-        return globby(e2eTestFiles);
-      })
-      .then(function (files) {
-        // gulp-mocha has an issue where, if there are no files passed from gulp.src,
-        // it will just run everything it finds
-        if (!files.length) {
-          return;
-        }
-        return mochaCmd();
-      })
-      .then(function () {
-        return cleanupSeq.reduce((promise, step) => promise.then(step), B.resolve())
-          .finally(function () {
-            if (opts.e2eTest.forceExit) {
-              process.exit(0);
-            }
-          });
-      }).catch(function (err) {
-        return cleanupSeq.reduce((promise, step) => promise.then(step), B.resolve())
-          .finally(function () {
-            env.spawnWatcher.handleError(err);
-          });
-      });
+    try {
+      // go through the steps to run the test
+      await startupSeq.reduce((promise, step) => promise.then(step), B.resolve());
```

Comment:
how about using bluebird?

Reply:
To what end? What would be the advantage?

Reply:
less code, better readability